waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/util/Whitelist.java

```diff
@@ -29,18 +29,18 @@
 
   public Whitelist() {}
```

Comment:
Seeing as you've looked at this class as part of this, what would the impact be of renaming this to `AllowList`? Would this be invisible to end users or does the YAML somehow refer to it? If we can rename this without any end user impact I think we should. We can also do that in a separate PR.

Reply:
It's invisible for the users so I'll change it.

Reply:
Should all the places which mention `whitelist` be changed to `allowList` too? Like `databaseToTableWhitelist` -> `databaseToTableAllowList`

Reply:
If this can all be done in a way that doesn't affect end users then yes please.